emperor/support_files/js/color-view-controller.js

```diff
@@ -356,6 +354,29 @@ define([
     return colors;
   };
 
+  /**
+   *
+   * Retrieve a whole discrete palette color set.
+   *
+   * @param {String} [map = 'discrete-coloring-qiime'] name of the color map to
+   * use, see ColorViewController.Colormaps
+   * @see ColorViewController.Colormaps
+   *
+   * @return {Object} map for selected color palette
+   *
+   */
+  ColorViewController.getDiscretePaletteColor = function(map) {
```

Comment:
From looking at the code, this method name should be changed to `getPaletteColors`. Based on line 374 this method could still retrieve a continuous colormap. Can you update the docstring, calls to this method, and unit tests?